asteroid/models/conv_tasnet.py

```diff
@@ -84,8 +85,9 @@ def __init__(
             )
         if causal and norm_type != "cgLN":
```

Comment:
In this case: `if causal norm_type not in [...]` would be better, so that we can also allow the channel normalization.